pkg/capabilities/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,62 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	cap "github.com/kubewarden/policy-sdk-go/pkg/capabilities"
+)
+
+type KubernetesHost struct {
+	cap.Host
+}
```

Comment:
As we are splitting the capabilities module into smaller ones. I think we need to create functions similar to the [`NewHost`](https://github.com/kubewarden/policy-sdk-go/blob/0b80f7587fc25a8f90aee1a8d9c80b516c16494c/capabilities/hostcaller_wasi.go#L21-L25). Therefore, we abstract away from the user the logic to properly create the host objects.